test/e2e/helpers/visu/PageTester.ts

```diff
@@ -68,30 +60,12 @@ export class PageTester {
   }
 
   /**
-   * @param fileName the name of the BPMN file without extension
+   * @param bpmnDiagramFileName the name of the BPMN file without extension
    */
-  private getPageUrl(
-    fileName: string,
-    bpmnLoadMethod: BpmnLoadMethod = BpmnLoadMethod.QueryParam,
-    loadOptions: LoadOptions = { fit: { type: FitType.HorizontalVertical } },
-  ): string {
-    log(`Use '${bpmnLoadMethod}' as BPMN Load Method for '${fileName}'`);
-
+  private getPageUrl(bpmnDiagramFileName: string, loadOptions: LoadOptions = { fit: { type: FitType.HorizontalVertical } }): string {
```

Comment:
⚠️ I suggest we find a better argument name for `bpmnDiagramFileName`
We are not expecting the extension so I will drop the 'file' part --> `bpmnDiagramName`?